### ISSUE TITLE
508 Accessibility compliance

### ISSUE THREAD

**Issue by gerardlopez2001** [QMDevTeam/QMDesigner#30]
For all user facing application features such as screens and menus, Section 508 Accessibility features should be enabled.

For more information, please see

http://www.section508.gov/IRSCourse/mod02/printVB.html

Related to QMDevTeam/QMGeneral#15
Related to #16 
Related to #17

**Comment by gerardlopez2001** [comment]
Moved to Trello: https://trello.com/c/GYdWQgBv
